aspnetcore/fundamentals/configuration/index.md

```diff
@@ -57,6 +57,35 @@ The following code displays the enabled configuration providers in the order the
 
 [!code-csharp[](index/samples/6.x/ConfigSample/Pages/Index2.cshtml.cs?name=snippet)]
 
+## Host configuration
+
+The following list contains host configuration sources from lowest to highest priority:
+
+1. `DOTNET_` prefixed environment variables
+1. Command-line arguments
+1. `ASPNETCORE_` prefixed environment variable
```

Comment:
Can we be consistent about providing links to `#evcp` and `#command-line` like we do for default application config sources. I want to make the list of sources look as much like a single list as possible while still making it clear that host bootstrap settings are only influenced by host configuration.